frontend/coreproject/src/lib/components/pages/anime_info/Index.svelte

```diff
@@ -1 +1,24 @@
-<p>Hello</p>
+<script lang="ts">
+  import { page } from '$app/stores';
+
+	import { anime_list } from '$data/mock/anime_list.ts';
+	import AnimeMainDetails from './AnimeMainDetails.svelte';
+
+  let anime_id = Number($page.params.id);
+
+	let anime = anime_list?.find((anime: any) => anime.id === anime_id);
+</script>
+
+<div class="anime_info overflow-hidden rounded-tl-3xl">
+	<div
+		class="anime_basic_details relative h-screen bg-cover"
+		style="background-image: url({anime?.bannerBackgroundImage ?? ''});"
+	>
+		<div
+			class="gradient absolute h-full w-full bg-gradient-to-t from-surface-500 to-surface-500/50"
+		/>
+		<div class="absolute h-full w-full md:p-16 2xl:p-28">
+			<AnimeMainDetails {anime} />
+		</div>
+	</div>
+</div>
```

Comment:
![image](https://user-images.githubusercontent.com/61817579/230700039-44500adb-ec53-4cea-855b-6de1e2eb75af.png)

![image](https://user-images.githubusercontent.com/61817579/230700060-dd6f6f0b-2c09-4140-95c7-021b61379a0b.png)

I will let you be the judge of this

Reply:
Hey! I'm now I'm fixing this
I only did for our devices

I need to do for devices between us, dw :)

Reply:
> Hey! I'm now I'm fixing this
> I  only did for our devices

> I need to do for devices between us, dw :)


Your approach is a bit flawed. But first fix the current problem. then i will address it.